FeedService/Models/PackageJson.cs

```diff
@@ -0,0 +1,52 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Text.Json.Serialization;
+
+namespace FeedService.Models
+{
+    public class PackageJson
+    {
+        /// <summary>
+        /// Represents the pilet name
+        /// </summary>
+        [JsonPropertyName("name")]
+        public string Name { get; set; }
+        public string Description { get; set; }
+        [JsonPropertyName("version")]
+        public string Version { get; set; }
+        public bool Preview { get; set; }
+        public string Custom { get; set; }
+        [JsonPropertyName("authoinfo")]
+        public Author AuthorInfo { get; set; }
+        public string Main { get; set; }
+        public string Lincense { get; set; }
+        public DevDependencies devDependencies { get; set; }
```

Comment:
These types (for `*Dependencies`) are wrong. They should `IDictionary<string, string>`. You could make an alias called `using Dependencies = IDictionary<string, string>`.